content/applications/inventory_and_mrp/inventory/barcode/operations/transfers_scratch.rst

```diff
@@ -1,17 +1,202 @@
+   in the database. If the same lot or serial number is shared between different products, scan the
+   product barcode first, then the barcode of the lot/serial number.
+
+Create a transfer from scratch
 ==============================
-Create a Transfer from Scratch
-==============================
 
-To create a transfer from the *Barcode* application, you first need to
-print the operation type barcodes. To do so, you can download the
-*Stock barcode sheet* from the home page of the app.
+In addition to processing and scanning barcodes for existing, previously-created internal transfers,
+the :guilabel:`Barcode app` can also be used to create transfers from scratch, simply by scanning a
+printed operation type barcode.
+
+.. admonition:: Did you know?
+
+   Odoo's *Barcode* application provides demo data with barcodes to explore the features of the app.
+   These can be used for testing purposes, and can be printed from the home screen of the app. To
+   access this demo data, navigate to the :menuselection:`Barcode app` and click :guilabel:`stock
+   barcodes sheet` (bolded and highlighted in blue) in the information pop-up above the scanner.
+
+   .. image:: transfers_scratch/transfers-scratch-demo-data.png
+      :align: center
+      :alt: Demo data prompt pop-up on Barcode app main screen.
+
+To do this, first navigate to the :menuselection:`Barcode app`. Once inside the *Barcode* app, a
+:guilabel:`Barcode Scanning` screen displaying different options is presented. From this screen,
+when using a USB or bluetooth barcode scanner, directly scan the product barcode. When using a
+smartphone as the barcode scanner, click the :guilabel:`Tap to Scan` button (next to the camera
+icon, at the center of the screen). This opens a :guilabel:`Barcode Scanner` pop-up screen that
+enables the camera of the device being used.
+
+Face the camera toward the printed operation type barcode to scan it. Doing so processes the
+barcode, and navigates to a barcode transfer screen.
+
+From this screen, an overview of all products to process within that specific internal transfer
+(**WH/INT/000XX**) is shown. Since this is a new transfer created from scratch, however, there
+should not be any products listed on the page.
 
-.. image:: transfers_scratch/transfers_scratch_01.png
-    :align: center
+To add products, click the :guilabel:`Add Product` button at the bottom of the screen, and add the
+products and product quantities that should be transferred. Once ready, click :guilabel:`Confirm`
+to confirm the changes made to the product line.
 
-Once done, you can scan the one for which you want to create a new
-document. Then, an empty document will be created and you will be able
-to scan your products to populate it.
+.. image:: transfers_scratch/transfers-scratch-blank-product-editor.png
+   :align: center
+   :alt: Blank product editor in scratch internal transfer.
```

Comment:
This new screenshot is really informative! I think we can further add to it by explictly mentioning how we navigate to the page, and directly referencing the image and the fields on the screen in an example admonition block. Feel free to ignore part of my comment though, as you included the navigational steps in the above paragraphs. I tend to include navigational steps again in an admonition block, since I assume ppl skim and miss info. 

```suggestion
.. example::
   To create an internal transfer directly from the :menuselection:`Barcode app`, navigate to the
   :guilabel:`Barcode Scanning` dashboard, and click the :guilabel:`Operations` button, then the
   `Internal Transfers` card.

   Then, click the :guilabel:`New` button, which opens a window with the reference number for the
   internal transfer (in this case, it's `WH/INT/00006`) at the top of the page. To add a product
   manually, click the :guilabel:`+ Add Product` button.

   Doing so opens a window to fill out product information for the transfer. (Todo: bring attention
   to the required fields and how to fill them out. I think the required fields are
   :guilabel:`Product`, :guilabel:`Unit of Measure`, :guilabel:`Source Location` and
   :guilabel:`Destination Location`.

   .. image:: transfers_scratch/transfers-scratch-blank-product-editor.png
      :align: center
      :alt: Blank product editor in scratch internal transfer.
```
![image](https://github.com/odoo/documentation/assets/23611711/2c6c4993-0d98-46ad-8611-3c0cd7a613e9)